web-project/src/test/java/com/epam/webproject/service/AuthorServiceTest.java

```diff
@@ -0,0 +1,53 @@
+package com.epam.webproject.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import com.epam.webproject.dao.AuthorDao;
+import com.epam.webproject.dao.BookDao;
+import com.epam.webproject.domain.Author;
+import com.epam.webproject.domain.Book;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration("classpath:dispatcher-servlet.xml")
+public class AuthorServiceTest {
+
+    @Mock
+    private AuthorDao authorDao;
+
+    @Autowired
+    @InjectMocks
+    private AuthorService authorService;
+
+    @Before
+    public void init() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void testGetByName() {
```

Comment:
I also suggest you to read about BDD practices like splitting the test into given/when/then parts, and shouldXXXwhenYYY test naming convention. You could practice on this particular method